src/components/LHNOptionsList/OptionRowLHNData.js

```diff
@@ -109,14 +93,6 @@ function OptionRowLHNData({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [fullReport, linkedTransaction, reportActions, personalDetails, preferredLocale, policy, parentReportAction, transaction]);
 
-    useEffect(() => {
-        if (!optionItem || optionItem.hasDraftComment || !comment || comment.length <= 0 || isFocused) {
-            return;
-        }
-        Report.setReportWithDraft(reportID, true);
```

Comment:
Here is a recording showing current (working) state of the feature when user is being added and then removed from the workspace:

https://github.com/Expensify/App/assets/62747088/3b804beb-53ef-48e7-b4be-c8d8530255f2

Reply:
@aimane-chnaif Unfortunately yes, I will try to find out what would be the better approach here - from initial investigations, it'd be nice if we, instead of updating entire list, just update a single item that is recipient of a draft. 

Let's keep it for the future